src/screens/Learn/Faq/FaqSection.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { Fragment } from 'react';
+import { StyledAccordion } from 'components/SharedComponents';
+import { SectionHeader } from './Faq.style';
+import { Section, Question } from 'cms-content/learn';
+
+const FaqSection = (props: { content: Section }) => {
+  const { content } = props;
+  const { sectionTitle, questions } = content;
+
+  return (
+    <Fragment>
+      <SectionHeader>{sectionTitle}</SectionHeader>
+      {questions.map((item: Question, i: number) => (
+        <StyledAccordion summaryText={item.question} detailText={item.answer} />
```

Comment:
Remember to add the key here (the ID is better)

Reply:
there's no id on the question, using `accordion-question-${i}` instead